features/ui/button/button.tsx

```diff
@@ -1,9 +1,45 @@
-import { ButtonHTMLAttributes } from "react";
+import React, { ButtonHTMLAttributes, ReactElement } from "react";
 import classNames from "classnames";
 import styles from "./button.module.scss";
 
-export function Button(props: ButtonHTMLAttributes<HTMLButtonElement>) {
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  size?: "small" | "medium" | "large" | "xlarge";
+  color?:
+    | "primary"
+    | "secondary"
+    | "gray"
+    | "empty"
+    | "empty-gray"
+    | "error"
+    | "empty-error";
```

Comment:
I prefer enums as they are easier to refactor. But I guess there are other opinions as well 😄